### ISSUE TITLE
NORMALISE_PARAMS problem

### ISSUE THREAD

**Issue by MihaiDogariu** [aasharma90/LightWeight_RefineNet#3]
Hello! I followed your code and the modifications that you brought from the NYU dataset implementation. I've noticed that you did not change the `NORMALISE_PARAMS` in the config file. This might affect the training. I found different values in another repo [here](https://github.com/fyu/drn/blob/master/datasets/cityscapes/info.json), namely:
```python
NORMALISE_PARAMS = [1./255, # SCALE
                    np.array([0.485, 0.456, 0.406]).reshape((1, 1, 3)), # MEAN
                    np.array([0.229, 0.224, 0.225]).reshape((1, 1, 3))] # STD
```
Can anyone else confirm this? This change would make sense, since these parameters are dataset-related.

**Comment by aasharma90** [comment]
Hi @MihaiDogariu, 

Yes, you are correct indeed. Actually, the params that you observe there are from the ImageNet dataset. You can change them according to the training dataset, but in my experience, I've not found any significant advantages in doing so. Some similar posts you can follow -

https://github.com/JiaRenChang/PSMNet/issues/81
https://github.com/speedinghzl/pytorch-segmentation-toolbox/issues/23
